src/darwin/Framework/CHIP/ServerEndpoint/MTRServerCluster.mm

```diff
@@ -39,6 +67,34 @@ @implementation MTRServerCluster {
     NSMutableSet<MTRAccessGrant *> * _accessGrants;
     NSMutableArray<MTRServerAttribute *> * _attributes;
     MTRDeviceController * __weak _deviceController;
+
+    std::unique_ptr<MTRServerAttributeAccessInterface> _attributeAccessInterface;
+    // We can't use something like std::unique_ptr<EmberAfAttributeMetadata[]>
+    // because EmberAfAttributeMetadata does not have a default constructor, so
+    // we can't alloc and then initializer later.  And we need a contiguous
```

Comment:
Wouldn't an `std::vector` work?

Reply:
I guess we could do that, as long as we ensure we don't mutate it after we hand the `data()` off to someone else.  Will do.

Reply:
Done.